app/src/main/java/ch/epfl/sdp/blindly/main_screen/MatchPageFragment.kt

```diff
@@ -270,4 +286,21 @@ class MatchPageFragment : Fragment(), CardStackListener {
         manager.setSwipeAnimationSetting(settings)
         func()
     }
+
+    /**
+     * Prepares the button to play an audio
+     *
+     */
+    private fun setupAudioPlayerButton() {
+        //val playAudio = fragView.findViewById<View>(R.id.play_audio_profile_button)
+        //playAudio.setOnClickListener {
+        //playAudio()
+        //Toast.makeText(context, "Ui", Toast.LENGTH_LONG).show()
+        //}
```

Comment:
Is it buggy for now ? I suggest you explain why it is commented.

Reply:
Oh I forgot to remove it ^^' I moved it somewhere else